desktop/extensions-sdk/build/set-up/frontend-extension-tutorial.md

```diff
@@ -157,9 +158,34 @@ A `metadata.json` file is required at the root of your extension directory.
 }
 ```
 
+The `title` property is the name of the extension that is displayed in the left-menu of the Docker Desktop Dashboard.
+The `root` property is the path to the frontend application in the extension's container filesystem used by the
+system to deploy it on the host.
+The `src` property is the path to the HTML entry point of the frontend application within the `root` folder.
+
+For more information on the `ui` section of hte `metadata.json`, see [Metadata](../../extensions/METADATA.md#ui-section).
+
+## Build the extension and install it
```

Comment:
I like that we have this build / instal step right in the flow here. Maybe we need to entirely remove the section about build / install in the TOC ?